frame/proxy/src/lib.rs

```diff
@@ -266,8 +265,9 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::remove_proxies(T::MaxProxies::get()))]
 		pub fn remove_proxies(origin: OriginFor<T>) -> DispatchResult {
 			let who = ensure_signed(origin)?;
-			let (_, old_deposit) = Proxies::<T>::take(&who);
-			T::Currency::unreserve(&who, old_deposit);
+
+			// Call the `remove_all_proxy_delegates` method to remove all proxies associated with
```

Comment:
this comment isn't useful at all

Reply:
fixed , will update in next push

Reply:
no need to comment to say you fixed it. push the code and Github will automatically pickup this code is changed.